src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,15 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery first = new Lottery();
+        first.getRandomBall();
+        Lottery second = new Lottery();
+        second.getRandomBall();
+        Lottery third = new Lottery();
+        third.getRandomBall();
+        Ball[] balls = new Ball[]{first, second, third};
+        for (Ball ball : balls) {
+            System.out.println(ball);
+        }
```

Comment:
![image](https://github.com/mate-academy/jv-lottery/assets/63095184/6f4f7563-6069-4900-b00e-e625b5e33c7a)

Reply:
it is written in ReadMe file - "To test the program, in the main() method, create three balls using the getRandomBall() method from the Lottery class. And print information about them in the console.". So why don't I need to create three balls and call getRandomBall method for each of them, if it is written for me to do so?